Contents/Code/__init__.py

```diff
@@ -100,18 +112,35 @@ def Update(self, metadata, media, lang, force, movie):
         flags = flags | Prefs['hideUsefulMiscTags'] << 3 #0b01000 : Hide Useful Miscellaneous Tags
         flags = flags | Prefs['hideSpoilerTags']    << 4 #0b10000 : Hide Plot Spoiler Tags
 
+        if movie:
+            series = HttpReq("api/serie/fromep?id=%s&level=3&allpics=1&tagfilter=%d" % (aid, flags))
+            movie_episode_data = HttpReq("api/ep?id=%s" % (aid))
+
+            aid = series['id']
```

Comment:
Best to use try_get in case an ID doesn't get returned

Reply:
Done